terraform/modules/alertmanager/input.tf

```diff
@@ -4,6 +4,17 @@ variable monitoring_space_id {}
 
 variable config { default = "" }
 
+variable slack_url { default = "" }
+variable slack_channel { default = "" }
+
+locals {
+  map_variables = {
```

Comment:
Can we make the name more explicit?
Like config_variables, template_variables, alertmanager_variables...